pkg/virt-controller/services/template.go

```diff
@@ -916,6 +922,15 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 				},
 			},
 		}
+		if selinuxType != virtconfig.DefaultSELinuxLauncherType {
```

Comment:
infra container also needs this (online 946)

Reply:
Switched to the loop you suggested, thanks!